src/ggrc/models/revision.py

```diff
@@ -711,6 +711,16 @@ def content(self):
 
     Updated by required values, generated from saved content dict."""
     # pylint: disable=too-many-locals
+
+    cached_content = None
+    if self.id:
+      cached_content = self.get_cached_content()
+
+    if cached_content:
+      content = self._content.copy()
+      content.update(cached_content)
+      return content
```

Comment:
Overall, the following would be more clear. Please rewrite this code.
```suggestion
    cached_content = self.get_cached_content()
    if cached_content is not None:
      return cached_content
```